examples/README.md

```diff
@@ -8,6 +8,10 @@ One or more pieces of ABL code which are self-contained and demonstrate a key fe
 
 - Concepts and features available in the ABL, from procedural to Object-Oriented examples.
 
+### OpenEdge Command Center
+
+- Demo scripts to setup an OpenEdge Command Center environment with support for OpenTelemetry metrics.
```

Comment:
Yeah, just leading off the README in the child folder with this would be helpful :)

Reply:
Added introduction paragraph to the README in the child folder.